tests/ignite/test_utils.py

```diff
@@ -174,6 +175,25 @@ def test_override_setup_logger(capsys):
     logging.shutdown()
 
 
+@pytest.mark.parametrize("encoding", [None, "utf-8"])
+def test_setup_logger_encoding(encoding, dirname):
+    if platform.system() == "Windows" and encoding is None:
+        return
```

Comment:
Actually, I wanted to check that we can't find test_words for this case. Do you think this can be possible to add ?